webdriver-spec.html

```diff
@@ -1244,6 +1244,12 @@
   time that commands will block before returning with a <a>timeout</a>
   error.
 
+  <p class=note>Navigation actions - Get, Back, Forward, Refresh - always,
+  unless they abort the navigation, complete with the <a>current top-level browsing context</a>
+  being the <a>current browsing context</a>. Even if we navigate Back or Forward to a state
+  where a different <a href=https://html.spec.whatwg.org/#browsing-context>browsing context</a>
+  was the <a>current browsing context</a>.
```

Comment:
What do you mean by “abort the navigation”?  I think this note is unnecessarily complicated if this is explained by the algorithms.